code/modules/projectiles/projectile/special.dm

```diff
@@ -202,34 +193,31 @@ obj/item/projectile/kinetic/New()
 /obj/item/projectile/plasma
 	name = "plasma blast"
 	icon_state = "plasmacutter"
-	damage_type = BURN
-	damage = 10
-	range = 6
-	var/power = 9
+	damage_type = BRUTE

```

Comment:
hacky fix to make it somewhat viable against asteroid mobs. Otherwise I'd have to buff its damage to 30 in unpressurized environment and I still think it'd be abusable (especially since all you need to do to get full power projectile is for the shooter to be in space.)